index.js

```diff
@@ -75,13 +76,17 @@ export default class WebTorrent extends EventEmitter {
     this.tracker = opts.tracker !== undefined ? opts.tracker : {}
     this.lsd = opts.lsd !== false
     this.utPex = opts.utPex !== false
+    this.natUpnp = opts.natUpnp || true
+    this.natPmp = opts.natPmp || true
     this.torrents = []
     this.maxConns = Number(opts.maxConns) || 55
     this.utp = WebTorrent.UTP_SUPPORT && opts.utp !== false
 
     this._downloadLimit = Math.max((typeof opts.downloadLimit === 'number') ? opts.downloadLimit : -1, -1)
     this._uploadLimit = Math.max((typeof opts.uploadLimit === 'number') ? opts.uploadLimit : -1, -1)
 
+    this.natTraversal = this.natUpnp ? new natApi({ enablePMP: this.natPmp }) : null
```

Comment:
as github suggests `NatAPI`, not `natApi`

Reply:
`this.natUpnp && new NatAPI` not `?`

Reply:
Both have been updated, thanks.